homework_1.py

```diff
@@ -11,5 +11,17 @@ def show_items():
     return f'<h3>This is a list of items: {constant}</h3>'
 
 
+@application.route("/add_item")
+def add_random_item():
+    constant.append(random.randint(1, 100))
+    return f"<h3>This function adds some value to the list: {constant}</h3>"
+
+
+@application.route("/delete_item")
+def delete_item():
+    constant.pop()
+    return f"<h3>This function removes the last element from the list: {constant}</h3>"
```

Comment:
One minor nit is that the message is not consistent, you show an updated list where the last element is already removed, it would be better to show the item that is removed or the initial list before it is updated.

Other LGTM!